pkg/controller/machine/controller.go

```diff
@@ -193,6 +193,35 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileMachine) create(ctx context.Context, machine *clusterv1.Machine) error {
```

Comment:
These functions were just removed. You should just remove this file from your commit.

Reply:
Whoops, fixing that!